python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py

```diff
@@ -74,6 +80,14 @@ def __init__(self, opsimdb, descqa_catalog, dither=True,
                                host='fatboy.phys.washington.edu',
                                port=1433, driver='mssql+pymssql')
 
+        self.agn_db_name = os.path.join(os.environ['SCRATCH'], 'proto_dc2_agn',
```

Comment:
We will want this to be an argument passed in by the user so that Tom can redirect to wherever he puts that AGN parameters database for the final run.  `test_agn.db` is just a cartoon database I created for testing the AGN `DESCQAObject` during development.